main.py

```diff
@@ -20,7 +20,7 @@ def init():
         "-d",
         "--data_path",
         type=str,
-        required=True,
+        default=r"C:\Users\turch\OneDrive\Documents\0CAAL\KG",

```

Comment:
```suggestion
        default=str(current_file_location / "data"),
```
Default data location should be relative to the location of the project. If we add the absolute data path, it is more likely to break.